GitExtensionsVSIX/GitExtCommandTable.cs

```diff
@@ -22,7 +22,7 @@ internal sealed class PackageGuids
     /// <summary>
     /// Helper class that encapsulates all CommandIDs uses across VS Package.
     /// </summary>
-    internal sealed class PackageIds
+    internal sealed partial class PackageIds
```

Comment:
why has this changed?

Reply:
Changed by Visual Studio, I let that automatic change pass

Reply:
Looks like a newer version of the tooling, from an earlier change in this file. Harmless enough.

Reply:
Nw, just checking.
I noticed that VS is modifying resources in the GitUI and after those automatic changes resources not longer build.